boards/shields/waveshare_epaper/waveshare_epaper.dtsi

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&arduino_spi {
+	status = "okay";
+	cs-gpios = <&arduino_header 16 GPIO_ACTIVE_LOW>,	/* D10 */
+		   <&arduino_header 12 GPIO_ACTIVE_LOW>;	/* D04 */
+
+	sdhc0: sdhc@1 {
+		compatible = "zephyr,mmc-spi-slot";
+		reg = <1>;
+		status = "okay";
+		label = "SDHC0";
+		spi-max-frequency = <24000000>;
+	};
+};
```

Comment:
For clarity this file would better be renamed as `waveshare_epaper_common.dtsi` or `waveshare_epaper_sdhc.dtsi`

Reply:
I will rename it to waveshare_epaper_common.dtsi

Reply:
That would be fine.